resources/lib/kodiutils.py

```diff
@@ -699,10 +708,10 @@ def get_cache(path, ttl=None):  # pylint: disable=redefined-outer-name
     if ttl is None or now - mtime < ttl:
         try:  # Python 3
             from json import loads, JSONDecodeError
-            ValueError = BaseException  # pylint: disable=invalid-name, redefined-builtin
+            ValueError = StubException()  # pylint: disable=invalid-name, redefined-builtin
```

Comment:
I don't think you have to instantiate exceptions.

Reply:
In fact I think this is wrong, because you catch Exception or Error classes not instances.

Reply:
In Python 3 I got:
`TypeError: catching classes that do not inherit from BaseException is not allowed`

Reply:
In that case you may have to do: `class ValueError(Error, BaseException):` ?

Reply:
Ah, Error does not even exist. My bad. `class ValueError(BaseException):`